src/studio/nuxeo-studio/nuxeo-studio-release-notes/2018-05-28-studio-73-13.md

```diff
@@ -0,0 +1,62 @@
+### Simplified Operations Configuration
+Configuring an operation in Studio Designer is now easier than ever thanks to several changes.
+- Default options are now compatible with the default automation chain created in Studio Modeler.
+- Icons are now suggested in the icon field
+![]({{file name='operations-icon-suggestion.png'}} ?border=true,w=344)
+- The input parameter field (used to send the initial input for the automation chain or script) is now checked upon save to make sure the value entered matches with the action slot you chose. If you use a wrong value, an error will be shown to guide you with the appropriate options:
+![]({{file name='operations-slot-input-check.png'}} ?border=true,w=650)
+Work is in progress to provide autocomplete while typing in the field.
+
+### Automated Document Type Label Translation Fallback
+When configuring a document type, a label is requested for it. If you do not create a translation file, Studio will now use this label as a fallback option to show your users something clearer.
+
+If you have already configured translations for your document type labels by uploading the corresponding files in Designer's **Resources** tab, they can be kept and will override the configuration made in the **label** field.
+
+### Simplified Theme configuration
+The theme editor in Studio Designer gained support for the left menu logo used in your theme using a dedicated option. Just upload it and you are done!
+
+Another option to upload a background image for your theme has been added as well. Note that this one requires you to configure how you want it displayed by adding a CSS entry in your theme configuration. First, upload the image you want; it will automatically be renamed to `background.png` in order to follow the Web UI conventions. Then, in your theme, add configuration for it:
+
+`--nuxeo-page-background: url(themes/myThemeName/background.png)`
+
+and you are done! Further CSS options like repeat and background color can be used as well.
+![]({{file name='theme-logo-background.png'}} ?border=true,w=650)
+
+### Save Your Studio Designer Code Changes Using Ctrl+S
+When using the "Switch to Code" option in Studio Designer, support for the `Ctrl+S` shortcut has been added so that you can save your changes anytime without leaving your keyboard.
+
+## Noteworthy Improvements and Bugfixes
+- When configuring a complex field in Studio Modeler, the generated label for these properties is now easier to read ([NXS-4606](https://jira.nuxeo.com/browse/NXS-4606)).
+- Studio Designer will warn you when an element for a float property is missing the attribute to allow entering non-integer values ([NXS-4625](https://jira.nuxeo.com/browse/NXS-4625)).
+- Using an aggregate on a multivalued complex property now generates valid configuration ([NXS-4462](https://jira.nuxeo.com/browse/NXS-4462)).
+- When losing authentication due to timeout, Studio will warn you everywhere ([NXS-4634](https://jira.nuxeo.com/browse/NXS-4634)).
```

Comment:
When authentication is lost due to a timeout, Studio warns you, no matter where you are in the application.